components/konflux-ui/production/kflux-prd-rh03/set-hostname.yaml

```
@@ -0,0 +1,4 @@
+---
+- op: add
+  path: /spec/host
+  value: konflux-ui.apps.kflux-prd-rh02.0fk9.p1.openshiftapps.com
```

This should point at rh03, not rh02.

```suggestion
  value: konflux-ui.apps.kflux-prd-rh03.nnv1.p1.openshiftapps.com
```